package.json

```diff
@@ -290,7 +290,7 @@
 		"chai": "4.2.0",
 		"chai-enzyme": "1.0.0-beta.1",
 		"check-node-version": "3.3.0",
-		"color-studio": "github:automattic/color-studio#1.0.1",
+		"color-studio": "github:automattic/color-studio#1.0.4",
```

Comment:
Looks like we could pull 1.0.5 directly from NPM :tada:

https://github.com/Automattic/color-studio/blob/master/CHANGELOG.md#105

Which changes were you interested in specifically?

Reply:
The earlier versions did not include the WooCommerce Purple / other brand colors. Thanks for pointing out this is on npm now! Updated.